src/Sarif.Driver/Sarif.Driver.csproj

```diff
@@ -37,7 +37,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="CommandLineParser" Version="2.7.82" />
+    <PackageReference Include="CommandLineParser" Version="2.9.1" />
```

Comment:
>Include [](http://example.com/codeflow?start=22&length=7)

OK, hm, I thought there was an API change to make. I guess not (or maybe it shows up in another client tool).

Reply:
Now I understand from the discussion, everything works, but you have to upgrade everyone at once. :)